gluten-substrait/src/main/scala/org/apache/gluten/backendsapi/BackendSettingsApi.scala

```diff
@@ -152,7 +152,7 @@ trait BackendSettingsApi {
 
   def needPreComputeRangeFrameBoundary(): Boolean = false
 
-  def supportCollectLimitExec(): Boolean = false
+  def supportCollectLimitExec(): Boolean = true
```

Comment:
Both the backends support it, let us remove this config

Reply:
done